dat/DOOM.dat

```diff
@@ -173,6 +173,20 @@ game (
 	rom ( name DOOM1.WAD size 4196020 crc 162b696a md5 f0cefca49926d00903cf57551d901abe sha1 5b2e249b9c5133ec987b3ea77596381dc0d6bc1d )
 )
 
+game (
+	name "Doom 64 Ex (v2.5)"
+	description "DOOM 64 EX (v2.5)"
+	rom ( name DOOM64.WAD size 8671616 crc 3b3e307d md5 e1f1c2bba2c1be41e22aa2e4d37d169e sha1 7656ddaa6d6ac405954d8b29946361a7114a1837 )
+	rom ( name DOOMSND.SF2 size 5429302 crc 03190b8e md5 51dfe2bd6d7f41c764a79893053cdffe sha1 e2fe47c504461cc88a683a798f2c9a20c43ce93a )
```

Comment:
Unfortunately multiple `rom` entries don't work with how libretro-database builds the database. Would you mind removing the DOOMSND entries? Thanks!
``` suggestion
```

Reply:
The SF2 and DSL files are actually Sound Fonts and are supported by some emulators, like zdoom, gzdoom, PrBoom Plus... I'm not sure about libretro-prboom though
More info:
https://zdoom.org/wiki/Sound_fonts